spec/wizards/placements/add_placement/steps/phase_spec.rb

```diff
@@ -0,0 +1,56 @@
+require "rails_helper"
+
+RSpec.describe Placements::AddPlacement::Steps::Phase, type: :model do
+  describe "attributes" do
+    it { is_expected.to have_attributes(school: nil, phase: nil) }
+  end
+
+  describe "validations" do
+    it { is_expected.to validate_presence_of(:school) }
+    it { is_expected.to validate_presence_of(:phase) }
+
+    context "when validating the phase" do
+      let(:school) { build(:placements_school) }
+
+      context "and the phase is primary" do
+        it "is valid" do
+          step = described_class.new(school:, phase: "Primary")
+
+          expect(step).to be_valid
+        end
+      end
+
+      context "and the phase is secondary" do
+        it "is valid" do
+          step = described_class.new(school:, phase: "Secondary")
+
+          expect(step).to be_valid
+        end
+      end
+
+      context "and the phase is not primary or secondary" do
+        it "is invalid" do
+          step = described_class.new(school:, phase: "Nursery")
+
+          expect(step).not_to be_valid
+        end
+      end
+    end
```

Comment:
Looking at the commit history on this branch, I can see you've already refactored these tests. If you wanted to, you could remove even more 'noise' and repetition from each context by doing this:

```ruby
describe "phase" do
  subject { described_class.new(school: build(:placements_school), phase:) }

  context "when the phase is primary" do
    let(:phase) { "Primary" }

    it { is_expected.to be_valid }
  end

  context "when the phase is secondary" do
    let(:phase) { "Secondary" }

    it { is_expected.to be_valid }
  end

  context "when the phase is neither primary nor secondary" do
    let(:phase) { "Nursery" }

    it { is_expected.to be_invalid }
  end
end
```

But I don't feel strongly about it – what you have already is good!